py/torch_tensorrt/_compile.py

```diff
@@ -61,17 +61,17 @@ def _get_target_ir(module_type: _ModuleType, ir: str) -> _IRType:
     else:
         if ir == "default":
             # Options are listed in order of preference
-            if module_is_fxable:
-                logging.log(
-                    logging.Level.Info, "ir was set to default, using dynamo as ir"
-                )
-                return _IRType.dynamo
-            elif module_is_tsable:
+            if module_is_tsable:
                 logging.log(
                     logging.Level.Warning,
                     "Input graph is a Torchscript module but the ir provided is default (dynamo). Please set ir=torchscript to suppress the warning. Compiling the module with ir=ts",
```

Comment:
This warning will show even when the input is an `nn.Module`, effectively making the default `ts`, since passing in `nn.Module` types is the common way to use Dynamo. Consider changing `module_is_tsable = any([module_type == t for t in [_ModuleType.nn, _ModuleType.ts]])` to `module_is_tsable = (module_type == _ModuleType.ts)` or similar.

Reply:
fixed the messaging